--- kmplayer-031113/kmplayer/src/kmplayer.desktop.orig	2003-11-13 19:20:16.000000000 +0100
+++ kmplayer-031113/kmplayer/src/kmplayer.desktop	2003-11-13 20:05:15.000000000 +0100
@@ -25,6 +25,6 @@
 Name[sr]=Kmpalyer
 Name[sv]=Kmplayer
 Name[xx]=xxKMPlayerxx
-MimeType=audio/x-mp3;audio/x-vorbis;application/x-mplayer2;application/x-ogg;audio/x-mp2;video/mpeg;audio/x-mpegurl;audio/x-wav;audio/x-mod;video/x-avi;video/x-ms-asf;video/x-ms-wmp;video/x-ms-wmv;video/x-msvideo;video/quicktime;video/x-flic;
+MimeType=audio/x-mp3;audio/x-vorbis;application/x-ogg;video/mpeg;audio/x-mpegurl;audio/x-wav;audio/x-mod;video/x-ms-asf;video/x-ms-wmp;video/x-ms-wmv;video/x-msvideo;video/quicktime;video/x-flic;
 InitialPreference=7
 Categories=Qt;KDE;AudioVideo
--- kmplayer-031113/kmplayer/src/kmplayer_component.desktop.orig	2003-11-13 19:20:16.000000000 +0100
+++ kmplayer-031113/kmplayer/src/kmplayer_component.desktop	2003-11-13 20:04:13.000000000 +0100
@@ -21,7 +21,7 @@
 GenericName[xx]=xxKMPlayerxx
 X-KDE-Library=libkmplayerpart
 # Keep in sync with the audio types in kmplayer.desktop
-MimeType=audio/x-mp3;audio/x-ms-wma;audio/x-vorbis;application/x-mplayer2;application/x-ogg;audio/x-mp2;video/mpeg;audio/x-mpegurl;audio/x-wav;audio/x-mod;video/x-avi;video/x-ms-asf;video/x-ms-wmp;video/x-ms-wmv;video/x-msvideo;video/quicktime;video/x-flic;
+MimeType=audio/x-mp3;audio/x-ms-wma;audio/x-vorbis;application/x-ogg;video/mpeg;audio/x-mpegurl;audio/x-wav;audio/x-mod;video/x-ms-asf;video/x-ms-wmp;video/x-ms-wmv;video/x-msvideo;video/quicktime;video/x-flic;
 Type=Service
 Icon=kmplayer
 ServiceTypes=KParts/ReadOnlyPart,Browser/View,KMediaPlayer/Player
